target/linux/ramips/image/mt76x8.mk

```diff
@@ -144,6 +144,18 @@ define Device/comfast_cf-wr758ac-v2
 endef
 TARGET_DEVICES += comfast_cf-wr758ac-v2
 
+define Device/cudy_lt500_v2
+  IMAGE_SIZE := 7872k
+  #IMAGES += factory.bin
+  #IMAGE/factory.bin := $$(sysupgrade_bin) | check-size | jcg-header 92.122
+  #JCG_MAXSIZE := 7872k
+  DEVICE_VENDOR := Cudy
+  DEVICE_MODEL := LT500 v2
+  #DEVICE_PACKAGES := kmod-mt76x2
```

Comment:
Delete any unnecessary comments.

Reply:
@obsy i'm still waiting to confirm that mt7615 is the right module for the MT7613B (no access to hardware atm.).

Reply:
look how the WR1300 v2 is setup (for the mt7613), it requires two or three packages for wifi, (mt7615e, mt7663-ap something and maybe a third one)

Reply:
@Djfe Thanks for the tip :pray: 
Sadly, I didn't pay attention to the revision when I was offered to keep hardware from another project. Before continuing the work on this PR, I need to buy the new revision and find some time.